src/shared/components/organisms/Sidebar/atoms/SidebarHeader.tsx

```diff
@@ -65,63 +55,10 @@ const SidebarHeader: AppComponent<SidebarHeaderProps> = ({
           path={mdiChevronDown}
         />
       </Button>
-      {controls != null && (
```

Comment:
We are using sidebar controls on the web application, and it entails everything from showing categories or the ordering of the categories.
Are you sure it does not impact anything there??